server/src/main/java/org/opensearch/repositories/Repository.java

```diff
@@ -198,6 +198,16 @@ default void deleteSnapshotsAndReleaseLockFiles(
      */
     long getRestoreThrottleTimeInNanos();
 
+    /**
+     * Returns restore throttle time in nanoseconds
+     */
+    long getRemoteUploadThrottleTimeInNanos();
```

Comment:
This won't break any plugin implementation, right?

Reply:
No it won't as `BlobStoreRepository` is implementing it. Also you should see the plugin as `incompatible` in the compatibility checker